### DISCUSSION TITLE
The external image is not visible on the Enterprise Portal Engine page.

### DISCUSSION THREAD

**Discussion by arulselvan008** [pimcore/pimcore#15245] [Q&A / Support]
In my product class, I have added the canto URL attribute to the data type "external image".
The attribute now includes a sample image URL, and I can access the image on the data object page.
I couldn't find the image on the PXP page. 

I've attached my configuration for your convenience. (Image URL :-https://imglarger.com/Images/before-after/ai-image-enlarger-1-before-2.jpg)

Products Class :- 
![image](https://github.com/pimcore/pimcore/assets/106297450/d9285356-0187-47e6-8b63-d8bf5f89055a)

Data Object (Id =240):-
![image](https://github.com/pimcore/pimcore/assets/106297450/2dd58e8c-4a9c-4560-b983-4d7ed6b01384)

PXP Page:-
![image](https://github.com/pimcore/pimcore/assets/106297450/e4048a86-0211-4c3c-a23c-94a4e9b51387)

I have debugged the PXP Page 
API :-
![image](https://github.com/pimcore/pimcore/assets/106297450/1c5fc670-4af7-43bd-a8b4-0ce3cb17c2e0)

Response Sample:-
![image](https://github.com/pimcore/pimcore/assets/106297450/233541bf-7672-4ad6-9033-149c372d846d)
![image](https://github.com/pimcore/pimcore/assets/106297450/01d89a7b-e664-496a-b320-68059211a51e)

Anybody can help us here?

**Comment by arulselvan008** [comment]
Add the default style of your image to the HTML file.

/vendor/pimcore/portal-engine/src/Resources/views/layout/default.html.twig

like :
 .data-type--external-image .media__item img{
    min-height:250px !important;
    min-width: 250px !important;
}

**Reply by lukmzig** [reply]
@arulselvan008 thank you for reporting this issue.
We had a look and we were unfortunately unable to reproduce this issue. If this issue persist I would kindly ask you to try to reproduce this behavior on our public demo. 
If that´s possible and you will manage to reproduce this, please feel free to open a Github issue with the steps how to reproduce it and we will have a closer look.
